core/src/main/java/io/kestra/core/runners/RunContext.java

```diff
@@ -35,6 +35,9 @@
 
 @NoArgsConstructor
 public class RunContext {
+    // List of env variables that should be redacted from the execution run context variables to avoid information disclosure.
+    private static final List<String> REDACTED_ENV_VAR = List.of("KESTRA_PLUGINS_PATH");
```

Comment:
add a configuration with a default value please

Reply:
I don't think it's a good idea that a user can update this list as we shouldn't help him to shoot him in the foot.

But this can be a different functionality to allow filtering env variables.

Reply:
we can since we have this lib : https://junit-pioneer.org/docs/environment-variables/

Reply:
I think there good use case (like a pod that expose sensitive data), just a matter of `@Value` to bring more feature, we definitely should do that